### PR TITLE
Update error-response-codes.md

### PR DESCRIPTION
**Pull request by letzya**
## Preview
https://deploy-preview-2510--tyk-docs.netlify.app/docs/nightly/error-response-codes/

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/error-response-codes.md [hunk 1]
```diff
@@ -10,7 +10,6 @@ menu:
 ---
 
 Tyk gateway responses include HTTP status codes that follow the [HTTP status code standard](https://datatracker.ietf.org/doc/html/rfc9110). They have three digits that describe the result of the request and the semantics of the response.
```
Comment (andyo-tyk) [inline tyk-docs/content/error-response-codes.md]:
```suggestion
Tyk Gateway responses include HTTP status codes that follow the [HTTP status code standard](https://datatracker.ietf.org/doc/html/rfc9110). They have three digits that describe the result of the request and the semantics of the response. 
```

File: tyk-docs/content/error-response-codes.md [hunk 2]
```diff
@@ -19,7 +18,10 @@ The first digit defines the class of response as shown in the [list](https://en.
 - 5xx (Server Error): The server failed to fulfill an apparently valid request
 
 ## Tyk Gateway error status code
-The error status codes (4xx and 5xx) used by Tyk Gateway are defined in the table below:
+
+Below is a list of error status codes (4xx and 5xx) along with their corresponding messages that may be returned by the Tyk gateway.
```
Comment (andyo-tyk) [inline tyk-docs/content/error-response-codes.md]:
```suggestion
Here we provide a list of all the error status codes (4xx and 5xx) that may be returned by the Tyk Gateway along with their corresponding messages and some guidance on the likely cause of the error. 
```

File: tyk-docs/content/error-response-codes.md [hunk 3]
```diff
@@ -19,7 +18,10 @@ The first digit defines the class of response as shown in the [list](https://en.
 - 5xx (Server Error): The server failed to fulfill an apparently valid request
 
 ## Tyk Gateway error status code
-The error status codes (4xx and 5xx) used by Tyk Gateway are defined in the table below:
+
+Below is a list of error status codes (4xx and 5xx) along with their corresponding messages that may be returned by the Tyk gateway. 
+If you require customised message for certain HTTP error codes, Tyk also supports [error templating]({{< ref "advanced-configuration/error-templates" >}}).
```
Comment (andyo-tyk) [inline tyk-docs/content/error-response-codes.md]:
```suggestion
Tyk supports [error templating]({{< ref "advanced-configuration/error-templates" >}}), allowing you to configure the Gateway to return customised messages for certain HTTP error codes.

We also support limited customisation of the error codes and messages returned by custom authentication middleware through the use of [override messages]({{< ref "tyk-oss-gateway/configuration#override_messages" >}}.
```

File: tyk-docs/content/error-response-codes.md [hunk 4]
```diff
@@ -19,7 +18,10 @@ The first digit defines the class of response as shown in the [list](https://en.
...
+
 | Code | Text                                                                                    | Description                                                                                                                                                             |
```
Comment (andyo-tyk) [inline tyk-docs/content/error-response-codes.md]:
```suggestion
| Code | Text                                                                                    | Recommended action                                                                                                                                                             |
```

**Review by andyo-tyk — DISMISSED** [review]
Some suggestions - including added link to override_messages (you may not want to add this, as I'm not clear how this works vs error templating)

File: tyk-docs/content/error-response-codes.md (hunk 4 above)
Comment (letzya) [reply tyk-docs/content/error-response-codes.md]: good one!!! wanted to change this as well

File: tyk-docs/content/error-response-codes.md [hunk 5]
```diff
@@ -19,8 +18,13 @@ The first digit defines the class of response as shown in the [list](https://en.
 - 5xx (Server Error): The server failed to fulfill an apparently valid request
 
 ## Tyk Gateway error status code
-The error status codes (4xx and 5xx) used by Tyk Gateway are defined in the table below:
-| Code | Text                                                                                    | Description                                                                                                                                                             |
+
+Here we provide a list of all the error status codes (4xx and 5xx) that may be returned by the Tyk Gateway along with their corresponding messages and some guidance on the likely cause of the error. 
+Tyk supports [error templating]({{< ref "advanced-configuration/error-templates" >}}), allowing you to configure the Gateway to return customised messages for certain HTTP error codes.
+
+We also support limited customisation of the error codes and messages returned by custom authentication middleware through the use of [override messages]({{< ref "tyk-oss-gateway/configuration#override_messages" >}}.
```
Comment (andyo-tyk) [inline tyk-docs/content/error-response-codes.md]:
```suggestion
We also support limited customisation of the error codes and messages returned by custom authentication middleware through the use of [override messages]({{< ref "tyk-oss-gateway/configuration#override_messages" >}}).
```

**Pull request merged**